metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -356,7 +369,8 @@ public void replay(PartitionChangeRecord record) {
         brokersToIsrs.update(record.topicId(), record.partitionId(),
             prevPartitionInfo.isr, newPartitionInfo.isr, prevPartitionInfo.leader,
             newPartitionInfo.leader);
-        log.debug("Applied ISR change record: {}", record.toString());
+        String topicPart = topicInfo.name + "-" + record.partitionId();
```

Comment:
It's probably useful to log the topicId too?

Reply:
sorry. added.